packages/api/src/errors.js

```diff
@@ -27,19 +27,24 @@ export class HTTPError extends Error {
 
   /**
    * @param {Error & {status?: number;code?: string;}} err
-   * @param {{sentry: Toucan}} ctx
+   * @param {{sentry: Toucan, req: Request}} ctx
    */
-  static respond(err, { sentry }) {
-    const { message, code, status } = maybeCapture(err, { sentry })
-    return new JSONResponse(
-      {
-        ok: false,
-        error: { code, message },
-      },
-      {
-        status,
-      }
-    )
+  static respond(err, { sentry, req }) {
+    const { message, code, status } = maybeCapture(err, { sentry, req })
+    return status === 302
```

Comment:
Non blocking, super nitpicky so ignore me if this doesn't fit the code style, returning a ternary is always a little more difficult for me to personally read. I personally prefer making a single if statement and a default return. Super non blocking nitpicky stuff, feel free to disregard.